core/data-plane-selector/data-plane-selector-core/src/main/java/org/eclipse/edc/connector/dataplane/selector/core/DataPlaneSelectorExtension.java

```diff
@@ -46,9 +48,15 @@ public void initialize(ServiceExtensionContext context) {
         var strategy = new DefaultSelectionStrategyRegistry();
         strategy.add(new RandomSelectionStrategy());
 
+        dataPlaneSelectorService = new EmbeddedDataPlaneSelectorService(selector, instanceStore, strategy, transactionContext);
+
         context.registerService(DataPlaneSelector.class, selector);
         context.registerService(SelectionStrategyRegistry.class, strategy);
-        context.registerService(DataPlaneSelectorService.class, new DataPlaneSelectorServiceImpl(selector, instanceStore, strategy, transactionContext));
+    }
+
+    @Provider(isDefault = true)
+    public DataPlaneSelectorService dataPlaneSelectorService() {
+        return dataPlaneSelectorService;
```

Comment:
you're right... I will change this accordingly

Reply:
ok, it makes sense, at the end having it as default has no point because someone wants either to have a embedded data plane selector service or a remote one, will fix this accordingly